fixed_slots_m:
  - [-0.66,  -2.310, 55.0, 0.8, 0.9]  # 원래 [2.00, 0.60, ...]